00_Base/src/util/middleware/ResponseValidationMiddleware.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  getMetadataArgsStorage,
+  InternalServerError,
+  KoaMiddlewareInterface,
+  Middleware,
+} from 'routing-controllers';
+import { plainToClass } from '../util';
+import { Service } from 'typedi';
+import { Context } from 'vm';
+import { validate } from 'class-validator';
+import { validatedResponseParam } from '../decorators/ValidatedResponseSchema';
+import {
+  OcpiResponse,
+  OcpiResponseStatusCode,
+  OcpiRoutePrefix,
+} from '../../index';
+import { ResponseValidationError } from '../../exception/ResponseValidationError';
+
+@Middleware({ type: 'before' })
+@Service()
+export class ResponseValidationMiddleware implements KoaMiddlewareInterface {
```

Comment:
Might be nice to have a one liner javadoc describing the purpose of this middleware.

Reply:
I'm under the impression that it is to add validation to outgoing responses.

Reply:
added